test/Microsoft.NET.Build.Tests/GivenThatWeWantToPreserveCompilationContextForBuild.cs

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToPreserveCompilationContextForBuild : SdkTest
+    {
+        public GivenThatWeWantToPreserveCompilationContextForBuild(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [WindowsOnlyFact]
+        public void It_supports_copylocal_false_references()
+        {
+            var testProject = new TestProject()
+            {
+                Name = "CopyLocalFalseReferences",
+                TargetFrameworks = "net461",
+                IsSdkProject = true,
+                IsExe = true
+            };
+
+            testProject.AdditionalProperties.Add("PreserveCompilationContext", "true");
+            testProject.PackageReferences.Add(new TestPackageReference("NETStandard.Library.NETFramework", "2.0.0-preview2-25330-01", null));
```

Comment:
The functionality from the `NETStandard.Library.NETFramework` package is going to be merged into the SDK (see #1267).  Once it does I don't know what will happen with this test that depends on the package that's no longer used.  It might fail, or it might silently stop giving you the test coverage you want.

Is there another way to test this without depending on this package?  Could you just build a project and then reference it's output DLL directly?